econet_base.yaml

```diff
@@ -154,6 +154,32 @@ text_sensor:
     request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
+  - platform: template
+    name: "Microcontroller Datetime"
+    id: irtc_time
+    icon: mdi:calendar-clock
+    update_interval: 30s
+    entity_category: "diagnostic"
+    disabled_by_default: true
```

Comment:
What's the reason for this being disabled by default? All the data points below are always requested anyway.

Reply:
My worry was that since this value is constantly changing it will create lots of history/logging entries in HASS. Is there a way to suppress these from logging/history?